src/diffpy/pdfgui/gui/dummyfile.py

```diff
@@ -1 +1,2 @@
-#Hi Simon Billinge. This is Beric's dummy branch. 
+#Hi Simon Billinge. This is Beric's dummy branch.
+# Making some secondary edits here. 
```

Comment:
Made some new edits here.

Reply:
nice.  Please don't forget to log this by responding to the previous comment though....